packages/documentation/src/stories/components/tab/tab.stories.tsx

```diff
@@ -21,18 +21,17 @@ export default {
 
 const Template = () => (
   <PostTabs>
-    <PostTabHeader slot="tab">First</PostTabHeader>
-    <PostTabPanel slot="panel">
+    <PostTabHeader slot="tabs" panel="first">First</PostTabHeader>
+    <PostTabHeader slot="tabs" panel="second">Second</PostTabHeader>
+    <PostTabHeader slot="tabs" panel="third">Third</PostTabHeader>
```

Comment:
Can you explain why you are using tabs in plural, please?

Reply:
Because it is the slot where all the tabs are: 

![image](https://github.com/swisspost/design-system/assets/33580481/a05314f1-3fb8-434a-bfb0-18ea46f9e2dd)